pom.xml

```diff
@@ -44,6 +44,11 @@
             <artifactId>structs</artifactId>
             <version>1.19</version>
         </dependency>
+        <dependency>
+            <groupId>org.jenkins-ci.plugins</groupId>
+            <artifactId>jackson2-api</artifactId>
```

Comment:
Curious as to why a new dependency needs to be introduced.

Reply:
github-api-plugin depends on this plugin, so it's not really adding a new dep, but the released version is very out-dated (causing enforcer errors).

If someone wants to release github-api-plugin then this dependency change is unneeded

Reply:
jenkinsci/github-api-plugin#35 is going to remove this dependency. Hopefully that gets released and you can just depend on the new version. @bitwiseman are you planning a new release soon?

Reply:
right, there shouldn't be a conflict anymore and then this dependency could be removed

Reply:
@res0nance 
I'm planning to release as soon as I reasonably can, in the meanwhile. I'll take this PR as it stands.

Reply:
We want this as of #269 anyway.